### ISSUE TITLE
Coins Coins Coins repatable Quests make the game to easy.

### ISSUE THREAD

**Issue by Dream-Master** [GTNewHorizons/NewHorizons#3189]
Coins Coins Coins repeatable Quests make the game to easy.
What we can do against it or what we can changed in code?

Disable repeatable quests at all. (possible)
Every time you buy this item it got more and more expensive (need code changes)
Make it repeatable x times before you cant it craft another time (5 times or so) (need code changes)
Disable Parties to make the game not so cheaty if players are in different tiers and don't play only buy items. (not sure if we can disable it or it need code changes)
Add a item to the repeatable quest that you can get over time to complete quest (need code changes additions)

I need you help with it to make GTNH hard again :P

**Comment by boubou19** [comment]
I'm in favor of the price increasing after each purchase because it will force players to make items by themselves instead of spamming quests in coins tab.

**Comment by damug** [comment]
Remove finished and intermediate products from coin buying section, leave raw materials/bees/etc to make up for rng. 

Instead allow people to purchase lootbags (possibly with different loot table [ie tier appropriate raw/intermediate materials only] from the currently available tiered bags) so there is still a relatively useful coin sink.

**Comment by Prewf** [comment]
I don't know if it's still a thing, but I remember it's possible to get free coins from repeatedly killing mobs from a repeatable quest in one of the tabs. There's one for killing skeletons, pigmen, creepers, etc. You could nerf that if you want to limit the supply of coins even further.

**Comment by richardhendricks** [comment]
Coins quests are not meant to replace gameplay, but only as occasional supplements when needed to get past a tight time.


Suggestions:

- [ ] Increase repeat time on any GT component quests to 2 weeks, and increase cost 5x
- [ ] Increase repeat time on ore quests to 1 week, and increase cost 5x
- [ ] One thing we can do is setup cross-quests like is done with the Thaumcraft node-in-a-jar.  I do not know how capable the BQ system is, but maybe set it up so that if you buy 1 rocket, you cannot buy *any* rocket for 4 weeks.  That way you cannot have a large group spam rockets and grind them right away for next tier.
- [ ] Look into how quest requirements/rewards are handled for teams and rewards.  Determine if rewards are per team (ok) or per member (problematic). If each member has to fulfill requirements, add in some "party gates" by consuming expensive items. If only 1 member has to fulfill requirements, we might have to add feature to disable parties to BQ or limit their size to 2 or 3 members, or find alternate ways to handle it.

We can't balance the game for both 1 player and 6 player teams.  I think it is fair to balance for 2-3 people working together, but not make the experience bad for single players. Large parties are just cheating themselves.

**Comment by richardhendricks** [comment]
I think the mob-killing quests are OK, since you have to physically do them, and only 1 person in the party gets credit, it limits how much they can do. I am OK with investing time -> coins.

The bigger problem is the large # of coins from quests in general, esp. in steam/lv/mv where there are many quests. They can construct 1 mixer and pass it around the whole party.  That's why adding some "consume items" quests will help with that (needs investigation).  Yes it would suck to lose an item you just made, but it prevents some of the cheating from parties.

**Comment by WarlordWossman** [comment]
I don't see why killing 10 mobs should give you technician coins (maybe no coins at all?).
It might take time but once you make coins farmable the items you can buy get really cheap. I don't think the whole system should be viewed as a "shop" area where you constantly buy things instead of making them yourself. The components I get even if they should be more expensive but raw materials I would just remove from there.
A stack of stainless steel is way too usefull to just hand it out for a few coins imo. In fact I would remove most of the quests for the ingots to encourage finding a stable source of the materials instead.

Parties will always get more, I wouldn't disable them since they make coop not feel like shit.
No team would want to do every crafting task 2 times, that just feels like we don't want people to have fun.

**Comment by richardhendricks** [comment]
Need to investigate `      "simultaneous": false,` tag, maybe forces only 1 reward for a party.  This would fix some of the issues, but "informal parties" (not a party in BQ) could still pass around machines etc. Would need to make it consume/craft item to progress.  IDK, it would be a hard thing to prevent completely.

**Comment by HegemonRast** [comment]
I am not so sure about the repeatable mob quests, killing ice queen over and over for orange coins is a bit much.  Maybe the lower mob quests are good but the twilight progression quests should not be repeatable for coins, (carmanite etc) maybe.    I agree with all these comments  I am most inclined towards the materials only  as the coins have become a way to avoid playing the game, if it was materials you could use it if you got stuck could not find nickel  but you would still have to otherwise play the game.  As it is now parties of 4 on our server are skipping tiers almost entire by buying mats and items.

**Comment by richardhendricks** [comment]
I hope the simultaneous tag is what we need to prevent parties from scumming the quests. They could still do it by splitting up their party though unless we add in crafting quests or consume quests. Increasing the timers will also help, but again we can't really balance for 1 player and 4 player teams.

**Comment by WarlordWossman** [comment]
Now that we have the ability to find veins it would be a better way to make more single ores available so people can search for the veins. A single ore will never really boost your progression too much and we could always choose one of the weaker ones of each vein.

The problem with a stack of ingots is that you can make a lot of things from it, it's too diverse to just hand that out. Rather give people one of the ores from a nickel vein and have them search for a new one using the scanner.

**Comment by richardhendricks** [comment]
Damn, simultaneous is not what we need either. Haha this clown! :) https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/2577119-better-questing-a-new-and-improved-questing-mod?page=5#c99

**Comment by Azzyj** [comment]
I think richardhenricks'  4 suggestions above hit the mark for me.  The only thing I'd like to add is I feel that you should be buying base materials only.  Buying a Rocket seems incongruous with the difficulty level of this pack, (especially when multiples are being issued on 1 purchase).

**Comment by richardhendricks** [comment]
The reason purchaseable rockets were added was because buggy rockets were leaving players, and we wanted a way to let them purchase a replacement rocket if their first rocket disappeared.

It looks like there is an option
`https://github.com/Funwayguy/BetterQuesting/search?q=partySingleReward&unscoped_q=partySingleReward`

This might be the one we need investigating.

**Comment by 0lafe** [comment]
I think that having coin quests for things you can find in world is fine, and that's about it. I like the Pam's ones, and the bee ones, because finding them in world just sucks. I don't think there should be ones for things like motors, conveyors, etc...

**Comment by richardhendricks** [comment]
I like having the motor etc quests because otherwise the coins just pile up and don't seem useful. I do agree they should be for emergencies or JIT use, not every time you need a new machine use.

**Comment by 0lafe** [comment]
Realistically we don't need coins for rewards at all. They're nice to have, but the game is perfectly playable without them, and more than doable. It just means you have to craft all the stuff yourself, and find things in the world.

**Comment by richardhendricks** [comment]
Might be possible to allow only a specific # of repetitions to a quest.

**Comment by richardhendricks** [comment]
I disagree, on the server it can be hard to find rare items since everything is plundered pretty fast.

**Comment by 0lafe** [comment]
Yea, I'd like to have a way to avoid that, but you can just add crafting recipes for some of this stuff, or give it as quest rewards

**Comment by sthegreat** [comment]
I agree with 0lafe, coins should only be usable for rare worldgen items. Right now they are too powerful because you can use them without ever setting up permanent production for some of the items, since you generally only need to make a few for the quest to unlock them in the shop. The chemical and ingot ones are especially powerful in my opinion because you can get a whole stack of diesel, lubricant, oil, iron 3 chloride, aluminium, etc. for usually just a small amount of coins.

I would recommend we remove the more powerful ones and nerf the others by making them significantly more expensive as they are bought more (maybe exponential/percent based if possible, like everything gets 40% more expensive with each purchase).

**Comment by CalterOK** [comment]
In my opinion, most of the coins received on a certain dash should be spent to open the next one, taking into account the number of participants in the group (if possible) ~ 90% * the number of participants.

**Comment by NeroOneTrueKing** [comment]
I hope that the option to purchase **Damascus Steel Ingots** stays. It's useful to make GT meta-tools, which break horribly often in low tiers, and it doesn't help the player with tier goals (making machines, machine parts, multiblock structures, etc).

Super-important that the reward can only be claimed by one person in the party -- ex 100 coins -> 2 parts
Right now, everyone in the party can claim the purchase even though only one person paid -- ex in a party of three, 100 coins -> 6 parts
So a party of 3 isn't getting 3x the amount of stuff, they're actually getting 9x (3x the coins from their quests, and then 3x as many parts per coin), which is what really needs to be fixed

**Comment by richardhendricks** [comment]
Ok, I didn't know that.  I think we can definitely look into fixing this,
that's a major exploit.  Will look tonight.

**Comment by Kortako** [comment]
I agree with what @richardhendricks said, that the coin system should be used as an "emergency" support system, and I think it does a pretty good job at doing that already, **provided that you are not in a party**.  Remember when buying GT components could be done at the current progress tier, and they were cheaper, AND they were on like a 40 second cooldown?  Now THAT was OP.

But as Nero up above confirmed, parties seem to be the problem, and that's what should be fixed.  Please don't make single player parties a slog just because it's OP in a party, fix party exploits instead.

**Comment by mitchej123** [comment]
I'm not in a party, and I like them as they are.  They're not needed, and in my estimation not OP; but they're a very nice quality of life for a one off crafting.  Generally if I want something I'll go craft a huge batch of it; the coins are nice for a few one off machines I want to build and I don't want to go source all the parts for an emitter or pump or something.   (yah yah, close to AE2... will get autocrafting soon).

A bunch of people all getting rewards for one person doing it.. in a party... does seem a bit OP and that should be addressed.

**Comment by noobyaran9** [comment]
Is there an option that scales the item according to the amount of player in team? Smth like 1.5^(amount of player). This way the item trading gets exponentially expensive with alot of player in a team which kinda make sense.

**Comment by richardhendricks** [comment]
OK, so our quest file uses a legacy format, that does not support the PARTY_LOOT variable.  It is set by default to false in api\properties\NativeProps.java. I am OK with setting this to `true`.  Another option is to add it as a config option in LegacyLoader_v0.java that can be used per-quest. 

In general, I think it should be OK to force it true globally, *however* the only instance I can see this being a problem is if a team has warp with every teammate, but in general I think the tear quest is "repeatable enough" even for a team.

The bigger concern is exploiting the coins.  Even with this fix, players could not "officially" party and get multiple coinsets, but at least they would not be able to pay once for everyone to get the reward from the Coin Coins Coins quests.

**Comment by richardhendricks** [comment]
@Dream-Master 
I see we have fork here, https://github.com/GTNewHorizons/BetterQuesting
but there is no tag for what we are currently shipping, and I can't get it to build since it has various problems.  I did not see it on our Jenkins server either, so I don't know how to get it working.

The change needed is at

https://github.com/GTNewHorizons/BetterQuesting/blob/1.7.10/src/main/java/betterquesting/api/properties/NativeProps.java#L46

Change that line to read true, and try a new build with 2 people in party and see if both can still claim rewards.

**Comment by Dream-Master** [comment]
if you work on it use alphest fix branch. we using this @richardhendricks

**Comment by richardhendricks** [comment]
OK, I added the single party loot option as an overall option to the default quest database (  "partySingleReward": true).  If someone on a private server wants to cheese, they can disable it.

I need some testers, both single player and  small parties - I tested with an SSP game and with 2 people, seemed to work fine.

https://github.com/GTNewHorizons/BetterQuesting/pull/2

Since this is a change to a very important mod, I would prefer a lot of testing before we just throw this on the server.

**Comment by richardhendricks** [comment]
Be sure to delete/remove the old version in your mods directory.

[BetterQuesting-1.0.173e_unofficial.zip](https://github.com/GTNewHorizons/NewHorizons/files/2143730/BetterQuesting-1.0.173e_unofficial.zip)

**Comment by DV666** [comment]
I will test it if you want Richard :) I am agree with this option, pretty abusive for a hardcore modpack to get multiples rewards, didn't like it.

About this topic, i just began the modpack few weeks ago and with my friends (we are only 3 on dedicated server), we earned a LOT of coin... and we didn't finish chapter 2 yet xD 
We have +3.000 technician coin T1.

I like the idea to increase cost of coins for each purchase : for me, this chapter is like a "bonus" or "joker" : better questing can help you to get some items but you can't spam it and need more coins for next purchase :)

I didn't see this chapter like a shop, just a helper.

Very impatient to see next changes :)

PS : Sorry if it's not clear, i am baguette :D

**Comment by richardhendricks** [comment]
Thanks, please test the updated BQ mod with your group.

There is currently no configuration setting to increase costs with the coins quests.  We can implement this via a quest chain with non-repeatable quests ending in a very expensive repeatable quest, but that would take a lot of time and effort.

Also, please look at and comment on https://github.com/GTNewHorizons/NewHorizons/issues/3206

**Comment by richardhendricks** [comment]
Yeah, we probably should not have TF boss killing quests repeatable, they get cleaned out pretty quick.

**Comment by richardhendricks** [comment]
- [ ] Get numbers on how many coins are in each tier quest tab
- [ ] Decide on how many items of a specific tier a player at that tier should be able to purchase
- [ ] Adjust prices appropriately
- [ ] Look at rewards from Kill all the things, align with tiers

**Comment by 0lafe** [comment]
>Yeah, we probably should not have TF boss killing quests repeatable, they get cleaned out pretty quick

Please don't, that's how you get tons of coins late game for the coin requirement quests. Just remove the dumb coin rewards and they're not an issue.

**Comment by richardhendricks** [comment]
> Please don't, that's how you get tons of coins late game for the coin requirement quests. Just remove the dumb coin rewards and they're not an issue.

You do realize how stupid this sounds?  If anything we should change either the quests that grant coins or the quests for the late game coins requirements.

**Comment by Hakameda** [comment]
just remove the machine parts, cells, ingots, garbage. The rest of it is mostly fine. Add things for servers like bosses to help ease resets/Silverwoods/maple and fucking lavender

**Comment by DV666** [comment]
Ok so i test your BQ version @richardhendricks yesterday and that's works perfectly :) 

I claim reward but my friend cannot (quest go complete directly when i earn loot). Very nice :p

I will inform you if i see some bugs about that.

**Comment by Dream-Master** [comment]
ok thanks @CalterOK

**Comment by Dream-Master** [comment]
@Bluebine @boubou19 @damug @Prewf @richardhendricks 
What is the essence now? Add more coins per machine part quest and more repeat time ?

**Comment by damug** [comment]
That might work to balance them out for now, change the time to every 2-3 hours, 2x or 3x the current cost.  That coupled with the edits @richardhendricks made to group quest rewards should turn them back into a quick something when you are too lazy to craft it yourself, rather than a way to skip entirely making components through a tier.

**Comment by Dream-Master** [comment]
I would say make it 5x more expensive and 10 times longer do no easy ride anymore.

**Comment by Dream-Master** [comment]
and yes I will put new version in where parties get only one time the loot each quest.

**Comment by Dream-Master** [comment]
@richardhendricks 
i mean a duct tape solution for next release i am working on it and will be out in the next days.

**Comment by Hakameda** [comment]
Just remove them all together, it's silly for this

**Comment by 0lafe** [comment]
https://www.strawpoll.me/16015453/r

I made a poll to see what people are thinking at this point

**Comment by Hakameda** [comment]
Thing is, you can't trust the majority of people usually. It's best to stick to the theme. Its a hardmode gregtech pack. It's not meant to be easy. If people whine for it to be easier, it's probably not the right pack for them anyway.

On Wed, 4 Jul 2018 at 18:16, 0lafe <notifications@github.com<mailto:notifications@github.com>> wrote:

https://www.strawpoll.me/16015453/r

I made a poll to see what people are thinking at this point

**Comment by Dream-Master** [comment]
@0lafe i missing in the poll the option replace full motors only with materials you can make machine parts with.

**Comment by Azzyj** [comment]
I thought this thread was intended to see what people are thinking, now there's a poll?  Not all things need consensus, we've been discussing this for 10 days let's just make a decision.

**Comment by HegemonRast** [comment]
I have looked at the pole  ... I don't have a twitch account or whatever and have tried to register but it wont let me.  Anyway, this thread has had 18 participants, of which I don't think any one has said to leave as is.  On this pole with 19 votes so far the majority say leave as is, that seems a bit odd.  Its meant to be hard,  I agree with Hakameda entirely,  we play this pack because its hard.  Now instead of using a nether star and radon, both requiring effort to get, I can spend 6 coins and get an iv sensor or emitter. Its not enough I can make nether stars with salis mundus, now I can buy quantum stars for next to nothing.  The way this is going, why not just add the ability to buy OP for 10 coins ... can we stop the craziness please.  If you want an easy pack there are thousands, pick one and play it stop trying to add this pack to that endless list.

**Comment by 0lafe** [comment]
I agree with you hege, I don't like what coins can but either and I want to roll back a lot of the coins coins coins changes. I was just interested in what people thought more than anything. It's surprising that players of a hardmode pack want it to be made easier

**Comment by Kortako** [comment]
It's not that people like @mitchej123 or I want to make the game _easier_, we want it to stay the _same_. That's not the same thing as "make the game easier" at all.  I feel like I've gotten in this argument before on some other issues, but saying that we think it's fine the way it is does **not** mean that we're just noobs that think everything should be easy-peasy.  We understand that things should have a level of difficulty to them, that's why we're playing this pack in the first place.  That's why I've been playing this pack for 8 months.  I _don't_ want everything to be easy, but sometimes I don't agree with just "nerf everything because we can" either. Coins have been in there at least since 2.0.0.0, and they haven't allowed me to just skip past all the progression, as if that's what you guys imply on this thread.  

I personally just think your approach is heavy-handed.  If Dream thinks that Coins are OP, and we think that's it's fine (completely ignoring the party bug, because that obviously needs to be fixed), then there obviously has to be some middle ground.  Saying "oh obviously we should remove them all" is not really a middle ground, it's extreme in the opposite direction.

Even though I think coins are fine the way they are, obviously a ton of people don't think it's alright, so what's wrong with just making the refresh times way longer and nerfing the prices a little?  Okay, so you can get an emitter for a bunch of coins once a day or week, that isn't going to make a difference in things.  This pack is all about automation, which means you need TONS of components to be able to do anything meaningful.

Honestly, I don't even care that much anymore, do whatever you want with the coins.  Ultimately it's up to Dream.  I'm mostly just annoyed at the echo-chamber here that's forming with the "truly hardcore" people saying "oh, you shouldn't trust the majority, they're obviously wrong".  What's the point in me voicing my opinion if you're going to completely discount it like that?

**Comment by Prewf** [comment]
I'm all for keeping it the way it is too, I was one of the people who voted that on the poll. If you're not pooling coin rewards (aka cheating), I think it's fine the way it is. Who is realistically sitting around every 40-60 minutes waiting for a cooldown off these quests so they can get a free motor to progress? At that sort of rate, you'd never get anywhere meaningful in a reasonable amount of time. 

I saw in 2.0.4.6 that an EV-tier of coin rewards was added. I think it should stop at HV, the way it was before. EV is around the time people should start delving into AE anyways, so there shouldn't be any handholding via coins past this point. But **before** that, I think it's fine the way it is. The cooldown is long enough where it's not abusable (unless you're in a party, which should 100% be fixed) and it's just enough where you can use it to get an occasional motor or pump because, hey I'm too lazy to craft one, for this one machine I need, just this one time. Almost like an "emergency" motor/pump if that makes sense. That sort of convenience is nice to have imo.

If you guys are absolutely deadset on nerfing it, after fixing the party issue, I'm ok with an increasing coin cost, as long as there is a max.

**Comment by mitchej123** [comment]
I agree with the last two posts.  What's game breaking about it?  A minor convenience for 1-2 items if you need to craft one thing.  Anything else you need in large numbers and using coins for it would be ludicrous. 

Same people (who are way past anything you'd get from these) that are consistently saying nerf the earlier game!

Fix the party exploit. Done.

**Comment by WizardSleve280** [comment]
Like the previous people have said before me, I also agree with keeping the coins the way they are, but with nerfing it for parties. I'm not trying to be redundant and repeat what has been said already but if the point of the modpack is to automate things (seeing as there is a way to just automate direcrafting table recipes) being able to buy one motor a day or however the cooldown is would not "ruin the hardcore feel". It would just be a small time saver over the long run of making a complete factory to process your ores or your epoxid production.

As for fixing parties, the only people that should be able to earn/spend coins should be the leader of the party. That way, no matter who is doing what, only one person will be earning coins and spending them. Trading coins would be a whole other issue though

**Comment by DoomSquirter** [comment]
late in the game but I always saw the rewards as a carrot and stick balance.

stick is obvious.  every nerf/hurdle that ppl have to go through to accomplish something in gtnh that they normally wouldn't in other packs.  and that list is pretty extreme.  (note this is not an indictment of said sticks/nerfs, it's just an assessment of quantity).

now where are the carrots?  by default, gt's own carrot is literally the tiers of machines and except for some rare exceptions (macerator for one), there is no tangible benefit for the higher tier machines except they are faster.  obviously you end up being able to do different machines, multi's as well so let's call that another carrot.  making cool contraptions and setting up neat processes etc.  but all of these really aren't as tangible as the nerfs are since the nerfs/sticks are ever present.

the quests tho.  you get a bag.  you get some free stuff...  hell, you get some free stuff that gives you a slight edge over some of those nerfs.  that's an extremely emotional type of carrot in that it's a gift, even if it's one of each sapling.  free is free.

imho, I like how the reward bags are setup and the coins quest, etc.  tbh, I've used them to buy motors etc when I didn't feel like making any.  it was an instant respite of "do I want to do 10 minutes of putting together stf and making these things or can I get some instant gratification right now".  

as far as worrying about ppl joining parties to pool the bags, selling them, etc.  anyone that does that ruins the game for themselves.  anyone that buys them is a sucker?  how does that affect anything or anyone else tho?  does the sapling count have a finite number that is reflected by the sudden upswing in ppl carrying loaded sapling bags?   I assume on the officials there could be a lot that I do not have knowledge of so I could be making light of a bigger problem.

**Comment by 0lafe** [comment]
>Same people (who are way past anything you'd get from these) that are consistently saying nerf the earlier game!

I'm an MV player now, and I'd like to be treated like it 

>the quests tho. you get a bag. you get some free stuff... hell, you get some free stuff that gives you a slight edge over some of those nerfs. that's an extremely emotional type of carrot in that it's a gift, even if it's one of each sapling. free is free.

Honestly, I love the quests and how they work. I'd hate to see them nerfed (except maybe the ores from the sifter quest). 


Yes, coin quests are a quick and easy way to get something, but I don't really see a difference between that and just being like, "I'm in LV now, I might as well spawn in myself some motors to make this one annoying machine". 

If people really want this in the game, I guess I'll stop complaining, but I really don't think it fits the theme of the pack.

**Comment by draknyte1** [comment]
So, I made these guys, who can substitute in for certain quests.
If I give them a wide enough trade list, their trades become even less exploitable.
![unknown](https://user-images.githubusercontent.com/3060479/42369181-848c032a-814c-11e8-8ff9-17fd3c2af3c8.png)

**Comment by Dream-Master** [comment]
@draknyte1 if you give me a config option to adjust trade list would be nice to balance it. 
But in addition i will make the repeat time on Motors and co a bit longer and more costy.
EV and IV components i will redo.

**Comment by sthegreat** [comment]
Can we also nerf chemicals and ingots down to 16 at a time, instead of a full stack?

**Comment by CalterOK** [comment]
In my opinion, the components above the EV are not worth adding to the purchase, because they require very complex components, and at this stage it is already possible to create an autocraft.

**Comment by 0lafe** [comment]
If logistic pipes gets added, and you can autocraft to your heart's content at MV, can we remove MV+ componenets from the rewards?

**Comment by Hakameda** [comment]
EA games presents Gregtech New Horizons. Sick of grinding for your components or having to put in any real effort? Fuck it, just buy it!

**Comment by mitchej123** [comment]
TediousCraft by Hakameda!  Thing's don't get easier as you progress, just more tedious - keep clicking more!
*click click click click click*

**Comment by Hakameda** [comment]
@kortako The whole draw of this pack was exactly that. Hardcore gregtech themed pack, where the goal is to setup complex crafting lines to make your life easier.  Here is an excerpt from the modpack page.


Is this modpack for me?

Does the thought of earning that top-end gear over weeks or months appeal to you? The usual packs feel like they are over in 1 or 2 weeks since you get OP gear so quickly?

How about designing a chemical processing system to rival a real oil refinery, with multiple inputs creating dozens of outputs and intermediate products to finally make a bar of plastic, rubber, explosives or other top end gear?

Does the sound of machinery processing ore via crushing, washing, centrifuging, smelting in a blast furnace, and cooling off in a vacuum freezer tickle your brain?

Then this is the pack for you.

Your pick is not ready.

Welcome home.

**Comment by mitchej123** [comment]
Let's take a look at what's offered by the coins quest:

1) Some hard to find bees & hives that are more RNG than skill.  Doesn't seem OP to me. Pretty sure they're also gated by some other bee quests.
2) Prior tier motors, pumps, conveoyrs, etc.  You have to have worked past it to unlock this, it's 2 at a time and has a cooldown, zomg OP?
3) A few sapplings, a few seeds.  Most of which are more RNG than skill.  Doesn't seem too OP to me.
4) Some spawn eggs for bosses -- A bit OP possibly, but when all the bosses are killed on the servers... I can see why.
5) A few ingots.  Oh no, I bought them instead of finding my damascus steel in a village.  Also has a cooldown and gated.
6) A few low tier wires.  Game breaking? Probably not
7) Vis Shroom - Not as RNG,honestly probably easier to just go find a bunch of these. 
8) A few wafers, diodes, capacitors, etc.  Nothing in quantities big enough to be game breaking.
9) Rockets - Ok I can see an issue with these, but they're gated past having already been there and it seems to be for bug loss replacement (GC is pretty damn buggy).
10) A few ore berries & essence berries.  Whoop, a bit less RNG so you can plant them instead of running around hoping RNG lands your way (how is that skill again?) or that someone else hasn't already picked them.

Having the option of these available is nice.  You've done work to get the coins, so spend them.  That's what you do with cash IRL.  The only ones I've actually used are the piston or pump ones when I'm looking to make a quick machine and I'm short one or two things and don't feel currently like going and making another batch of 64+ (Which I'll likely start crafting while I make my machine).

**Comment by Kortako** [comment]
@Hakameda When I put "truly hardcore" in quotes, I was implying that some of you seem to think you are more harcore than the rest of us, which I would argue is decidedly not true. I would argue that anyone who decides to stick with this pack has to be significantly more "hardcore" than any of the other people in the Minecraft community to enjoy this pack. We are all making a decision to dedicate months of our life on this pack, because what you quoted is exactly what we enjoy.  However, I don't see how that means we can't still argue for some balancing of things. You seem to want to tell us "agree with us or GTFO, this pack isn't for you", when we've been playing alongside you guys for months just fine.  I think that's really unfair.

If that's not what you were trying to say, then I apologize for misinterpreting, but then you are going to need to explain what you meant in more detail.

**Comment by mitchej123** [comment]
free: 1. without cost or payment.

Pretty sure these cost coins, thus not free!


Ok how about this -- You play without using them, and let the people who enjoy a break from the RNG keep them.

**Comment by Hakameda** [comment]
No, as I said earlier. World gen related ones are fine. Helps with servers
alot. It's the machine parts/cells that should be removed

0Lafe and I just started over, I know there OP cause we used them. Did the
calcs last night. We are at HV, only MV for quests and we could buy
something like 500 diesel cells with our current coins. Even half that with
1 person is a shit ton.

Buying Motors is not super useful, but robot arms, sensors, emitters are
redonk.

Tedium is different for everyone, we put additional strain on our progress
of using only rails. No pipes, No AE, Only annoying thing I had to do so
far was Twilight forest. Getting ready to push cleanroom and EV circuits now

Coins remove alot of crafting, it's not good like that. It should focus on
playing smart. Building systems for what you need

On Jul 6, 2018 11:28 AM, "0lafe" <notifications@github.com> wrote:

Some hard to find bees & hives that are more RNG than skill. Doesn't seem
OP to me. Pretty sure they're also gated by some other bee quests.

I don't think anyone is complaining about these, they're great.

Prior tier motors, pumps, conveoyrs, etc. You have to have worked past it
to unlock this, it's 2 at a time and has a cooldown, zomg OP?

Yeah kinda. It's giving you free, important/useful items lol. That's 2
motors you don't have to craft, and get for FREE.

A few sapplings, a few seeds. Most of which are more RNG than skill.
Doesn't seem too OP to me.

Again, not the complaint really, those are great, especially on servers,
which this pack seems to be meant for.

Some spawn eggs for bosses -- A bit OP possibly, but when all the bosses
are killed on the servers... I can see why.

I could go either way on these. As long as you dont end up with a net
profit of coins after turning in the kill quest, but also not really the
game breaking problem.

A few ingots. Oh no, I bought them instead of finding my damascus steel in
a village. Also has a cooldown and gated.

64 of one of the best ticon ingots, that needs an ebf, which you can buy,
cheap, when running the ebf for that long would be almost impossible,
especially because it needs tons of steel to even make the Damascus dust.

A few low tier wires. Game breaking? Probably not

Better than motors, but still the same problem. Why aren't we encouraging
people to craft these items, and progress themselves, instead of giving
them what they need. I get that it has a cooldown, but each machine doesn't
need much.

Vis Shroom - Not as RNG,honestly probably easier to just go find a bunch of
these

I had a bitch and a half of a time trying to find them on dirthuts. All the
magical forests I found were looted already. Ended up trading for them.

A few wafers, diodes, capacitors, etc. Nothing in quantities big enough to
be game breaking.

But enough to make circuits easier than otherwise. Again, you should have
to craft these.

Rockets - Ok I can see an issue with these, but they're gated past having
already been there and it seems to be for bug loss replacement (GC is
pretty damn buggy).

Except you can buy them, and break them down for parts for the next one,
which is a fairly large help. Free T1 rocket materials.

A few ore berries & essence berries. Whoop, a bit less RNG so you can plant
them instead of running around hoping RNG lands your way (how is that skill
again?) or that someone else hasn't already picked them.

These are the ones I think people want to keep in the game, people are that
it's not skill.

. The only ones I've actually used are the piston or pump ones when I'm
looking to make a quick machine and I'm short one or two things and don't
feel currently like going and making another batch of 64+ (Which I'll
likely start crafting while I make my machine).

I still don't see how that's different than you going to craft a machine,
realising you're a few items short, and spawning them in because you can't
be asked to make them. There isn't a clear answer, but it's kind of a stray
from the gtnh ideas of old. Giving away items can be fine, but I personally
wouldn't want it in here, as I don't think fits the theme of the pack in
the slightest.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GTNewHorizons/NewHorizons/issues/3189#issuecomment-403098314>,
or mute the thread

**Comment by Hakameda** [comment]
It's a themed pack, should stay on theme. I know some friends who don't
like gtnh cause it's too hardm. They would rather play BR and that's fine.
This is meant to be a challenge

**Comment by mitchej123** [comment]
Mixed replies:

> Buying Motors is not super useful, but robot arms, sensors, emitters are redonk.

I think I've done that once, mainly for a lower tier.  (shrug).  I'm fine if the price on those go up, they're helpful but not game breaking in my mind.

> We are at HV, only MV for quests and we could buy something like 500 diesel cells with our current coins. Even half that with 1 person is a shit ton.

Diesel isn't that hard... if you want to waste coins on that I don't personally have a problem with it.  I'd prefer to make it myself - so if you don't think it's appropriate - don't buy it; but it's nice to have as an option if you accidently run out, or blow up your base and need some emergency rations!


> Tedium is different for everyone

I agree, let people play how they want instead of forcing your version of how to play on everyone else.

> Coins remove alot of crafting

I disagree with the "a lot".  They remove a small bit unless you're milking them -- bigger timeouts help with that.

> Except you can buy them, and break them down for parts for the next one, which is a fairly large help. Free T1 rocket materials.

Now that sounds like a good reason to make them so you can't disassemble them/break them down.  

> I had a bitch and a half of a time trying to find them on dirthuts. All the magical forests I found were looted already. Ended up trading for them.

I hate RNG; tedium at it's finest -- Great reason to keep these in.


> I still don't see how that's different than you going to craft a machine, realising you're a few items short, and spawning them in because you can't be asked to make them. There isn't a clear answer, but it's kind of a stray from the gtnh ideas of old. Giving away items can be fine, but I personally wouldn't want it in here, as I don't think fits the theme of the pack in the slightest.

How are the loot bags different?  Stop doing quests, free items!  You've got the coin rewards.. they should be useful for something.  If you can't see the difference between cheating items in, and having them as a trade-able resource for other actions.. well then not sure what else can be said.

**Comment by mitchej123** [comment]
> It's a themed pack, should stay on theme 

It's on theme for me and several others.  Stop forcing your version of theme on everyone else.  Having a few niceties doesn't make it less of a hardcore pack; it makes it slightly less tedious every once in a while.

**Comment by mitchej123** [comment]
Which was initially brought up because of the cheaty party system - something being fixed.  The discussion is being had, we're voicing our opinions, you're voicing yours -- but you're also going a step further and saying "my vision is the right one", whereas we're saying -- "they're nice, we can tweak them a bit, but to each their own if they want to use them or not".

I'm also fine with some tweaking of them to fit the theme, but flat out removal is a bit extreme.

**Comment by Prewf** [comment]
>0Lafe and I just started over, I know there OP cause we used them. Did the
calcs last night. We are at HV, only MV for quests and we could buy
something like 500 diesel cells with our current coins. Even half that with
1 person is a shit ton.

What are those calcs? Those diesel quests give you 3 hours of an MV amp or ~45 minutes of an HV amp. And it's chemical coins (which are arguably a lot less in stock than Technician coins). You make it sound like it's a sustainable option to keep powering your base with these coins, when really it's not. Again, it is, by design, only allowing you to use them in a very limited fashion (both because of how many coins there are in fluctuation and how many cells you get from a quest). So honestly, it's in your best interest to use these as an "emergency" option, not a main power option for your base. But by all means, go ahead and "abuse" them; you'll be out of coins before a day or two just to prove a weak argument.

>Buying Motors is not super useful, but robot arms, sensors, emitters are
redonk.

Yes, they are redonk, if they go past HV (because sensors/emitters start requiring radon at EV). You also have to consider you have to be at tier (n+1) before you can actually use these things at tier (n), so you can't really abuse them **when you really want them**.

>Coins remove alot of crafting, it's not good like that. It should focus on
playing smart. Building systems for what you need

A lot of these quests have 20-40 minute cooldowns. How does that translate to "remove alot of crafting" in your mind? Again, unless you're pooling coins and rewards in a party, or abusing the party bug, there's nothing that needs to be fixed here.

>But that's what this pack was until recent times, the guy who makes it
acknowledges it's not quite what he thought it should be and opened a git
issue to discuss it

Dream isn't perfect (no offense @Dream-Master). He has a vision, but he needs people like us to keep him grounded and not make the pack go off to extremes. For example, the insane vacuum freezer nerfs a couple weeks ago. If it weren't for people like us making an issue about it, you'd have 1 liquid air cell taking 12 minutes because Dream thought that was fine. That wasn't fine for a lot of people, except for the few people who go through some insane mental gymnastics to convince themselves that it's fine (probably because "hey I went through tougher shit, you guys should deal with it too" attitude).

**Comment by Hakameda** [comment]
Niceties isn't part of a theme for this pack or else alot of stuff would
not be tweaked so much. Sounds more like you do want it to be just like
every other pack. Being too easy just makes it boring and alot less likely
to play long term. Like Heg said, perhaps just buy OP for some coins

**Comment by mitchej123** [comment]
> Niceties isn't part of a theme for this pack or else alot of stuff would not be tweaked so much. 
So sayeth you.

> Sounds more like you do want it to be just like every other pack. 
Uh, please show the logic that gives rise to that conclusion?  I don't think anyone here has said that.  

> Being too easy just makes it boring and alot less likely to play long term. 
You've yet to show how it's "too easy".  This is a little bit nicer, nothing game breaking.  Again, forcing your opinions of theme on everyone else.

**Comment by Hakameda** [comment]
70 coins for 30.7 mil eu with diesel

**Comment by Dream-Master** [comment]
ok addition to the things i wrote before i can lower the cells you get to 32 or 16. Maybe i make it depnding on the Level. Pre lv 64 lv 32 mv and obove 16 or so.

**Comment by Hakameda** [comment]
So sayeth the man in charge, cause there is literally nothing in this pack
that I've thought back and said golly gee this sure is nice to craft

**Comment by Prewf** [comment]
It's actually 307 million eu. That might seem like a lot but's only enough to power 1 mv amp for a day and a half. I'm fine with lowering it to 16 per quest reward, as long it's we're not completely removing the reward altogether.

**Comment by Prewf** [comment]
Also please don't let components go past HV. I don't think we need EV+ component as quest rewards. For the rocket coins I can see people "abusing" them by disassembling etc, but by the same logic as I used for the chemical coins, this is only doable once or twice simply because of the low number of coins in circulation (they would also be forgoing any rocket emergency options, which is fine. people need to have options). If coin costs went up as rocket tiers went up, that would help in further limiting potential abuse.

**Comment by Kortako** [comment]
> Sounds more like you do want it to be just like every other pack.

You really didn't read anything I posted, did you?  That's not what any of us are saying at all.  You're still doing it, being incredibly hyperbolic and saying that we aren't "real" players because we disagree with you.

> Niceties isn't part of a theme for this pack or else alot of stuff would not be tweaked so much.

Actually, niceties are a part of this pack, or else everything would be pure GT and nothing would ever get easier as you tiered up.  I guess we should remove AE2, because that's too nice?  Shouldn't we have to hand craft everything?  Maybe we should remove LCRs, because that's too easy not having to use cells? 
I can be hyperbolic too.

These "niceties" are introduced gradually as you tier up, and so are the coin quests.

> Being too easy just makes it boring and alot less likely to play long term.

@mitchej123 and I have been around for a very long time.  We use coins.  We don't think it's too easy, and I still plan to play for a very long time.  Even if Dream removes coins, I will be a little sad, but I won't stop playing because of it.  Coins for components aren't even going to be relevant anymore to me, because I'm approaching the point where I can set up AE.  But I appreciated that it was there before that point.  We think it's fine the way it is, but we're okay with compromising by going along with Dream's suggestions.  But you don't seem to be willing to compromise, do you?

**Comment by Hakameda** [comment]
The description of the modpack, is every recipe gregified. That's generally
not a nicetity ,

**Comment by Dream-Master** [comment]
Guys and Girls this is a place to discusse the issue not to fight.

**Comment by richardhendricks** [comment]
All your coin calcs will no longer apply with the next release.  Times are seriously increased, as well as prices.

We are also looking at implementing increasing costs for the coins quest too, so there will be lots of dials and levers to play with.

I agree the point of the pack is automation,  but I disagree that the component reward quests necessarily discourage that.  Getting components for a previous tier encourages automation IMO - It makes it easier to setup a rack of LV sifters or macerator or compressors for example, but a few free motors doesn't make the whole setup for free.

Maybe someone powerleveling tier quests has an advantage, but honestly,  getting to EV that way leaves your setup brittle as hell - 3 or 4 HV macerators, distillation towers, compressors, centrifuges, etc make your setup way more robust.

Let's collect info on coins per tier and costs per quest after the next release and then we can discuss rebalance again.
